docs/how_we_work/issue_review.md

```diff
@@ -0,0 +1,13 @@
+# Reviews Process for Issues
+
+As a team we will use labels for tracking which reviews are assign to whom and issue comments to tag our teammates when an issue is ready for their review.
+
+## Process
+
+- At least one developer and one PM as reviewers on an issue
```

Comment:
Suggest the following:
- Standard review process
  - One other dev and one PM as reviewers on an issue
  - Dev review followed by PM review
  - Option to use more or less review when appropriate (e.g., tricky or big changes might need more dev or PO eyes, fixing spelling or typos may require only one review)

Reply:
Thanks! Although, I'm not sure the second bullet is necessary on all tickets, is it? If review needs to be sequential maybe we should specify that in the original issue? Like this issue, for example, didn't require sequential review. 🤷‍♀️ Not sure!

What do y'all think @alexsoble @amymok

Reply:
Those bullets pulled from the [GH thread](https://github.com/18F/OPRE-Unicorn/issues/31#issuecomment-880290313).  The bulk of our work will be dev/coding work, not documentation, and the norm there is typically code review before PM review.  I imagine there will be exceptions, which could be explicitly stated in the 3rd bullet.

Reply:
I think we did talk about having the PM review follows the dev review.  My understanding is that the PM review for code work is more to let PM knows that the work is completed.

Reply:
Ok @ninamak issue addressed. Ready for re-review when you get a sec.
PS just in case you don't know, you won't be able to see the changes made in this view, you have to view the file again to see the changes.